importer/migrations/0001_squashed_0015_auto_20180925_1851.py

```diff
@@ -0,0 +1,328 @@
+# Generated by Django 2.0.9 on 2018-10-04 15:00
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    replaces = [
+        ("importer", "0001_initial"),
+        ("importer", "0002_auto_20180709_0833"),
+        ("importer", "0003_auto_20180709_0933"),
+        ("importer", "0004_auto_20180812_1007"),
+        ("importer", "0005_auto_20180816_1702"),
+        ("importer", "0006_auto_20180912_0229"),
+        ("importer", "0007_auto_20180917_1654"),
+        ("importer", "0008_campaigntaskdetails_project"),
+        ("importer", "0009_convert_project_text_to_keys"),
+        ("importer", "0010_auto_20180920_2013"),
+        ("importer", "0011_auto_20180922_0208"),
+        ("importer", "0012_auto_20180923_0231"),
+        ("importer", "0013_auto_20180924_1318"),
+        ("importer", "0014_auto_20180924_1943"),
+        ("importer", "0015_auto_20180925_1851"),
+    ]
+
+    initial = True
+
+    dependencies = [
+        ("concordia", "0019_auto_20180920_1503"),
```

Comment:
Should we change the three concordia dependencies to the squashed one now?

Reply:
Django recognizes that the squashed concordia dependency replaces any references to the dependencies it squashes. Which is why I had to remove the dependency in one of the non-squashed importer migrations - it errored out because it had been applied prior to the concordia squashed migration. So I think the answer is, we don't have to. I did deploy this  branch to AWS and tested it there with DBs that were pre-existing. I also tested it locally after wiping out my docker DB volume.

Reply:
Okay, we'll need to update this when we remove the old migrations but we can do that then since we'll need to update the concordia migrations, too.